patches/server/0962-Piglin-Dancing-API.patch

```diff
@@ -0,0 +1,67 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Amin <amin.haddou@frg.wwschool.de>
+Date: Wed, 22 Feb 2023 10:27:04 +0100
+Subject: [PATCH] Piglin Dancing API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPiglin.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPiglin.java
+index 48d0a4e42e1b90d1323784d1284acabfe9497dd6..c2d11d00eed5cb3c3c81e5e22af55d5318736e88 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPiglin.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPiglin.java
+@@ -1,9 +1,7 @@
+ package org.bukkit.craftbukkit.entity;
+ 
+ import com.google.common.base.Preconditions;
+-import java.util.Collections;
```

Comment:
we generally don't touch imports in patches, so please revert the changes here and use the fully qualified name for the MemoryModuleType class